tests/gcoap_fileserver/main.c

```diff
@@ -29,6 +29,7 @@ static msg_t _main_msg_queue[MAIN_QUEUE_SIZE];
 /* CoAP resources. Must be sorted by path (ASCII order). */
 static const coap_resource_t _resources[] = {
     { "/const", COAP_GET | COAP_MATCH_SUBTREE, gcoap_fileserver_handler, "/const" },
+    { "/vfs", COAP_GET | COAP_PUT | COAP_MATCH_SUBTREE, gcoap_fileserver_handler, VFS_DEFAULT_DATA },
```

Comment:
Only nitpick: Maybe fix the CI warning about this line being longer than 100 characters.
Else I trust you that it works well.